src/components/vote-list.js

```diff
@@ -0,0 +1,55 @@
+import React, { useState, useEffect } from "react";
+import styled from "styled-components";
+import axios from "axios";
+
+export default function VoteList({ name, voteCount, id, i, getCandidateList }) {
+  const Vote = () => {
+    axios
+      .put(process.env.API_HOST + "/candidates/" + id + "/vote/", {
+        params: {},
+      })
+      .then(function (response) {
+        console.log(response);
+        getCandidateList();
+        alert(name + "님께 투표 했습니다!");
+      })
+      .catch(function (error) {
+        console.log(error);
+        alert("투표 실패했습니다!");
+      });
+  };
+
+  return (
+    <Wrapper>
+      <Candidate>
+        <Rank>{i}위:</Rank> {name} [{voteCount}]표
+      </Candidate>
+      <VoteButton
+        onClick={() => {
+          Vote();
+        }}
+      >
```

Comment:
```suggestion
      <VoteButton
        onClick={handleVote}
      >
```
onClick에는 실행시킬 함수를 넣어주면 됩니다. ```handleVote()```는 따로 매개변수를 받지 않기 때문에 위와 같이 코드 작성해주시면 됩니다.